src/utils/pickContainerApp.ts

```diff
@@ -9,8 +9,9 @@ import { ext } from "../extensionVariables";
 import { ContainerAppItem } from "../tree/ContainerAppItem";
 
 // TODO: support creating a new container app from picker
-export async function pickContainerApp(context: IActionContext): Promise<ContainerAppItem> {
+export async function pickContainerApp(context: IActionContext, exclude?: string | RegExp | (string | RegExp)[]): Promise<ContainerAppItem> {
     return await azureResourceExperience<ContainerAppItem>(context, ext.rgApiV2.resources.azureResourceTreeDataProvider, AzExtResourceType.ContainerAppsEnvironment, {
         include: ContainerAppItem.contextValueRegExp,
+        exclude,
```

Comment:
This excludes items from being final picks, however in this case, the ideal experience is to exclude ingress disabled Container Apps from being ancestor picks, and final picks.

I think applying that change to the `exclude` property would be a breaking change. So I'm thinking about adding new ancestor-pick-specific filter options. They can be optional.